packages_rs/nextclade/src/tree/params.rs

```diff
@@ -14,13 +14,17 @@ pub struct TreeBuilderParams {
   #[clap(long)]
   #[clap(num_args=0..=1, default_missing_value = "true")]
   pub without_greedy_tree_builder: bool,
+
+  #[clap(long)]
+  pub masked_muts_weight: f64,
 }
 
 #[allow(clippy::derivable_impls)]
 impl Default for TreeBuilderParams {
   fn default() -> Self {
     Self {
       without_greedy_tree_builder: false,
+      masked_muts_weight: 0.5,
     }
   }
 }
```

Comment:
The weight param and its default are defined here

Reply:
Nice! I think we might want a lower default so that masks is only used to resolve otherwise identical séquences.